Doc/library/netrc.rst

```diff
@@ -12,6 +12,11 @@
 
 --------------
 
+This module implements an object-oriented interface to :file:`.netrc` files.
+
+VxWorks has no user home directory so it doesn't support ftp auto-login. This module
+is not supported on VxWorks.
```

Comment:
Minor: it seems wrong to make a note about a niche OS so high in the doc here.
Suggest pushing it a little below (just before the class documentation), maybe using an `availability` directive (example https://github.com/python/cpython/blob/6000087fe979705dc588a46a35da884cc0198c67/Doc/library/time.rst#functions ref https://github.com/python/cpython/blob/2d6097d027e0dd3debbabc702aa9c98d94ba32a3/Doc/library/intro.rst#notes-on-availability)

Reply:
@merwok Thanks for your review. Seems putting the note between the sentence a "The netrc class parses and encapsulates the netrc file format used by the Unix ftp program and other FTP clients" and sentence b "class netrc.netrc([file])" is not that correct because the sentence a is explaining the class netrc.netrc() other than the whole module. That is, this module is lack of module summary. So I add the availability as you proposed.